src/js/tracks/text-track-list.js

```diff
@@ -44,6 +44,10 @@ vjs.TextTrackList = function(tracks) {
 
 vjs.TextTrackList.prototype = vjs.obj.create(vjs.EventEmitter.prototype);
 vjs.TextTrackList.prototype.constructor = vjs.TextTrackList;
+// emulate attribute EventHandler support to allow for feature detection
+vjs.TextTrackList.prototype.onchange = null;
```

Comment:
What about looping over the `allowedEvents_` object to add these items so we have only one place these things get listed?